### ISSUE TITLE
Check types for UConvert.

### ISSUE THREAD

**Issue by dneto0** [KhronosGroup/SPIRV-Tools#902]
Blocked by Vulkan CTS removing invalid tests.  
See Khronos internal issue https://gitlab.khronos.org/Tracker/vk-gl-cts/issues/720 for removal of the offending tests.  See https://gerrit.khronos.org/#/c/1821

We need to wait until test updates are propagated to all live CTS branches

**Comment by dneto0** [comment]
Cc @atgoo

**Comment by dneto0** [comment]
@atgoo added code for this in source/validate_conversion.cpp  but it's turned off by #if since we're waiting for Vulkan CTS to be updated.

**Comment by dneto0** [comment]
https://gerrit.khronos.org/#/c/1821 has been merged into Vulkan CTS.
Can enable these tests now.

**Comment by dneto0** [comment]
Fixed by #1457
